en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -78,6 +78,124 @@ Here are some of the graphs generated in Choreo cloud.
 
 [![Choreo Analytics Latency]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)
 
+## ELK Analytics for Choreo Connect
+
+[![Choreo ELK Analytics Data Flow]({{base_path}}/assets/img/deploy/choreo-elk-analytics-data-flow.png)]({{base_path}}/assets/img/deploy choreo-elk-analytics-data-flow.png)
+
+Enforcer component in Choreo Connect can log analytics data to be used by Filebeat in ELK Stack. Data flow for the ELK Analitics can be depicted as given above.
+
+## Setting up
+
+### Step 1 - Configuring Choreo Connect
+
+#### Step 1.1 - Configuring the config.toml file
+Open the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/config.toml` file and update the analytics section according to the below configuration.
+``` toml
+[analytics]
+ enabled = true
+    
+[analytics.enforcer]
+[analytics.enforcer.configProperties]
+ type = "elk"
+```
+With this configuration, default reporter class (`org.wso2.am.analytics.publisher.reporter.elk.ELKMetricReporter`) will be enabled. If you want to use a custom reporter then you need to compile the custom reporter implementation as a JAR file and mount it to the `/home/wso2/lib/dropins` directory within the enforcer and set the `publisher.reporter.class` in the `config.toml` like below.
+
+```toml
+[analytics]
+ enabled = true
+    
+[analytics.enforcer]
+[analytics.enforcer.configProperties]
+ type = "elk"
+ "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
+```
+
+#### Step 1.2 - Enabling logs
+
+Update the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/log4j2.properties` file as described below:
+<br/>
+
+Log to console
```

Comment:
We can say that there are two options available and i.e. log to the console and log to a file.  Then list down these two options. Based on the selected option, you need to configure the filebeat agent.